intellij-plugin-structure/structure-intellij/src/main/java/com/jetbrains/plugin/structure/intellij/problems/LevelRemappingPluginCreationResultResolver.kt

```diff
@@ -9,11 +9,11 @@ import com.jetbrains.plugin.structure.intellij.plugin.IdePlugin
 import kotlin.reflect.KClass
 
 class LevelRemappingPluginCreationResultResolver(private val delegatedResolver: PluginCreationResultResolver,
-                                                 additionalLevelRemapping: Map<KClass<*>, PluginProblem.Level> = emptyMap()
+                                                 additionalLevelRemapping: Map<KClass<*>, RemappedLevel> = emptyMap()
   ) : PluginCreationResultResolver {
 
-  private val remappedLevel: Map<KClass<*>, PluginProblem.Level> = additionalLevelRemapping +
-    mapOf(ForbiddenPluginIdPrefix::class to PluginProblem.Level.WARNING)
+  private val remappedLevel: Map<KClass<*>, RemappedLevel> = additionalLevelRemapping +
+    warning<ForbiddenPluginIdPrefix>()
```

Comment:
Why not `ignore` and why not to add `TemplateWordInPluginId` also here as a default remapping? Otherwise, I think we shouldn't have default remapping at all and let the client define remappings.

Reply:
Removed any of the default remappings. Clients should decide which mappings with corresponding levels they should use.